src/app/(profile)/(components)/ContactForm/ContactForm.tsx

```diff
@@ -147,6 +149,15 @@ export default function ContactForm({
               </span>
             </CheckboxInput>
           </InputFormError>
+          <InputFormError error={formik.errors.captcha}>
+            <CaptchaCheckbox
+              id="captcha"
+              label="I am not a robot"
```

Comment:
**nit:** Let's change copy to `Confirm your humanity with a quick click`

Reply:
quite funny label 👀